File: WebSiteDatLich/src/main/java/com/example/WebSiteDatLich/model/Position.java
package com.example.WebSiteDatLich.model;

public class Position {
    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getPosition_id() {
        return position_id;
    }

    public void setPosition_id(Integer position_id) {
        this.position_id = position_id;
    }

    private Integer position_id;
    private String name;

}

File: ungdungdatlichkham/lib/models/Position.dart
class Position {
  int? positionId;
  String? name;

  Position({this.positionId, this.name});

  factory Position.fromMap(Map<String, dynamic> map) {
    return Position(
      positionId: map['position_id'],
      name: map['name'],
    );
  }

  Map<String, dynamic> toMap() {
    return {
      'position_id': positionId,
      'name': name,
    };
  }
}
